app/controllers/families_controller.rb

```diff
@@ -0,0 +1,36 @@
+class FamiliesController < ApplicationController
+  def index
+    @families = Family.all
+  end
+
+  def new
+    @family = Family.new
+  end
+
+  def create
+    family = Family.create(family_params)
+    if family.save
+      redirect_to root_path
+    else
+      redirect_to "/"
```

Comment:
We usually do `render :new` here, where we display validation errors.
We also usually set a flash message here when something goes wrong, so the user is not left wondering as to what happened.

Reply:
👍